sdk/core/core-tracing/src/interfaces.ts

```diff
@@ -260,7 +254,7 @@ export interface Span {
    *
    * @returns the SpanContext object associated with this Span.
    */
-  context(): SpanContext;
+  spanContext(): SpanContext;
```

Comment:
As per our offline discussion - consider if we even need to expose this at all if we abstract a bit more of our testing code.

Reply:
I do want to do this, but we need to abstract away our testing code first so I created https://github.com/Azure/azure-sdk-for-js/issues/15730 to track this.